captum/attr/_models/base.py

```diff
@@ -28,32 +28,42 @@ def __init__(self, embedding, full_name):
         self.embedding = embedding
         self.full_name = full_name
 
-    def forward(self, input, *other_inputs, **kwargs):
+    def forward(self, *inputs, **kwargs):
         r"""
          The forward function of a wrapper embedding layer that takes and returns
          embedding layer. It allows embeddings to be created outside of the model
          and passes them seamlessly to the preceding layers of the model.
 
          Args:
 
-            input (tensor): Embedding tensor generated using the `self.embedding`
-                    layer using `other_inputs` and `kwargs` of necessary.
-            *other_inputs (Any, optional): A sequence of additional inputs that the
+            *inputs (Any, optional): A sequence of inputs arguments that the
                     forward function takes. Since forward functions can take any
                     type and number of arguments, this will ensure that we can
-                    execute the forward pass using interpretable embedding layer
-            **kwargs (Any, optional): Similar to `other_inputs` we want to make sure
+                    execute the forward pass using interpretable embedding layer.
+                    Note that if inputs are specified, it is assumed that the first
+                    argument is the embedding tensor generated using the
+                    `self.embedding` layer using all imput arguments provided in
+                    `inputs` and `kwargs`.
+            **kwargs (Any, optional): Similar to `inputs` we want to make sure
                     that our forward pass supports arbitrary number and type of
-                    key-value arguments
+                    key-value arguments. If `inputs` is not provided, `kwargs` must
+                    be provded and the first argument corresponds to the embedding
+                    tensor generated using the `self.embedding`.
 
```

Comment:
It might be good to add a note here that the ordered dict behavior for kwargs which is relied on here is new in python 3.6 and not guaranteed to remain consistent for future versions based on this spec: https://docs.python.org/3/whatsnew/3.6.html#whatsnew36-pep468

Reply:
added